bootstrap/main.go

```diff
@@ -113,8 +113,11 @@ func mainImpl() {
 			raven.SetTagsContext(map[string]string{
 				"gke_clusterrolebindingError": err.Error(),
 			})
-			fmt.Fprintln(os.Stderr, "WARNING: For GKE installations, a cluster-admin clusterrolebinding is required.")
-			fmt.Fprintf(os.Stderr, "Could not create clusterrolebinding: %s\n", err)
+
+			errText := err.Error()
+			if strings.Contains(errText, "Forbidden") || strings.Contains(errText, "forbidden") {
+				exitWithCapture(opts, "Could not create clusterrolebinding. GKE role \"containers.admin\" required to create resources.\n%s\n", err)
```

Comment:
I think we should also use the  as in some part of the UI it's the only string visible:

Could not create clusterrolebinding. GKE role "containers.admin" (Kubernetes Engine Admin) required to create resources.

Reply:
I did it the other way around, made a bit more sense IMO, lgty?

Reply:
sure!